kuma/static/styles/components/compat-tables/bc-table.scss

```diff
@@ -0,0 +1,263 @@
+/*
+Table layout
+- uses display: block for smaller screen sizes;
+- requires .bc-table-{$num} modifier to display columns correctly
+********************************************************************** */
+
+.text-content table.bc-table { /* specificity to over-ride .text-content styles */
+    width: 100%;
+    table-layout: fixed;
+    border: none; /* to over-ride .text-content styles */
+    border-collapse: separate; /* to over-ride .text-content styles */
+    border-spacing: 0;
+    @include rgba-fallback('background-color', $table-blue, .5);
+}
+
+.bc-table,
+.bc-table thead,
+.bc-table tbody,
+.bc-table tr,
+.bc-table th,
+.bc-table td {
+    box-sizing: border-box;
+}
+
+/* tr
+====================================================================== */
+
+.bc-table tr {
+    position: relative;
+    border-bottom: 0 solid transparent;
+}
+
+/* th & td
+====================================================================== */
+
+.bc-table th,
+.bc-table td {
+    position: relative;
+    padding: 5px 3px;
+    border-color: $bc-border-color;
+    border-top: $bc-border-width-thin solid $bc-border-color;
+    border-right: 0;
+    border-bottom: 0 solid transparent;
+    border-left: $bc-border-width-thin solid $bc-border-color;
+    text-align: center;
+}
+
+
+/* no right border on empty cell in first row */
+.bc-table thead tr:first-child td {
+    border-right: 0;
+}
+
+/* first column, fixed at 200 */
+.bc-table thead td,
+.bc-table tbody th {
+    width: 200px;
+    min-width: 200px;
+    max-width: 200px;
+    overflow: hidden;
+}
+
+/* everything except first column, 12 columns */
+@for $num from 2 through 12 {
+    .bc-table-#{$num} thead th,
+    .bc-table-#{$num} tbody td {
+        width: (100 / $num) * 1%;
+    }
+}
+
+/* second column */
+.bc-table thead td + th,
+.bc-table tbody th + td {
+    border-left: 0;
+}
+
+/* thicker border between platforms */
+.text-content .bc-platform-border {
+    border-left: $bc-border-width-thick solid $bc-border-color;
+}
+
+
+/* th
+====================================================================== */
+
+.text-content .bc-table th {
+    background-color: transparent;
+    overflow: hidden;
+    padding: 5px 3px 5px 6px;
+}
+
+.bc-table .bc-platforms th {
+    width: auto;
+    padding: 5px 0 2px 0;
+    border-top: 0;
+    border-bottom: $bc-border-width-thick solid $bc-border-color;
+    font-size: 24px;
+}
+
+.bc-table .bc-browsers th {
+    border-top: 0;
+    padding: 3px 0;
+    font-size: 18px;
```

Comment:
Nit: Perhaps use `@include $note-font-size` - The descriptive name of the variable probably makes it seem odd though.